test/test_adapters/test_multi_sim.py

```diff
@@ -236,10 +242,25 @@ def test_world_multi_sim_in_5s(self):
         multi_sim.stop_simulation()
         self.assertAlmostEqual(time.time() - start_time, 5.0, delta=0.5)
 
+    # def test_apartment_multi_sim_in_5s(self):
+    #     viewer = MultiverseViewer()
+    #     multi_sim = MujocoSim(
+    #         viewer=viewer, world=self.test_urdf_2_world, headless=headless
+    #     )
+    #     self.assertIsInstance(multi_sim.simulator, MultiverseMujocoConnector)
+    #     self.assertEqual(multi_sim.simulator.file_path, "/tmp/scene.xml")
+    #     self.assertIs(multi_sim.simulator.headless, headless)
+    #     self.assertEqual(multi_sim.simulator.step_size, self.step_size)
+    #     multi_sim.start_simulation()
+    #     start_time = time.time()
+    #     time.sleep(5.0)
+    #     multi_sim.stop_simulation()
+    #     self.assertAlmostEqual(time.time() - start_time, 5.0, delta=0.5)
```

Comment:
🤔

Reply:
yeah I need a test with a urdf with meshes, but these meshes live in a ros package, which can't run in CI